### PR TITLE
Add namespace labels to update deployment

### PR DESCRIPTION
**Pull request by adam2k**
## Description

Allow for revision label to be update to deployment namespaces when there is a change to the deployment.

## 🎟 Issue(s)

Resolves astronomer/issues#1347

## 🧪 Functional Testing

During updating a deployment does the revision number label appear (relies on the matching branch for Houston API - in progress)

Required for the Houston API PR: https://github.com/astronomer/houston-api/pull/476

### CODE REVIEW

File: pkg/proto/commander.pb.go [hunk 1]
```diff
@@ -1,7 +1,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
 // 	protoc-gen-go v1.23.0
-// 	protoc        v3.11.4
+// 	protoc        v3.13.0
```
Comment (andriisoldatenko) [inline pkg/proto/commander.pb.go:4]: i think we can remove this change.

File: provisioner/kubernetes/kubernetes.go [hunk 2]
```diff
@@ -83,11 +83,25 @@ func (k *KubeProvisioner) InstallDeployment(request *proto.CreateDeploymentReque
...
 }
 
+// UpdateDeployment updates the deployment and optionally the deployment namespace values
 func (k *KubeProvisioner) UpdateDeployment(request *proto.UpdateDeploymentRequest) (*proto.UpdateDeploymentResponse, error) {
+	logger := log.WithField("function", "UpdateDeployment")
 	response := &proto.UpdateDeploymentResponse{
 		Deployment: &proto.Deployment{},
 	}
 
+	if request.NamespaceLabels != nil {
```
Comment (andriisoldatenko) [inline provisioner/kubernetes/kubernetes.go:93]: can we add tests or do you need help?

Comment (adam2k) [reply provisioner/kubernetes/kubernetes.go:93]: Yeah, I'm not sure if I'd need to write a test file for the k8s provisioner or if there is a different way to get coverage on this file.

**Review by sjmiller609 — APPROVED** [review]

**Review by andriisoldatenko — APPROVED** [review]

**Comment by adam2k** [comment]
@andriisoldatenko thanks for writing the tests! 🌮 🎉

**Pull request merged**